src/base/ApplicationEventBridgeRule.ts

```diff
@@ -39,13 +48,15 @@ export class ApplicationEventBridgeRule extends Resource {
       throw new Error('AWS allows only up to 5 targets per event bridge rule');
     }
     const eventBus = this.config.eventBusName ?? 'default';
+    const { scheduleExpression, eventPattern } = this.config;
```

Comment:
do we need to explicitly check in code that one of the above is defined? if neither are defined, will this fail to deploy?

on that note, should we add a test when neither are provided?

(not sure how restrictive/guardrail-y we want to be here.)

Reply:
checking if it will synth

Reply:
The error is thrown when you attempt to plan or validate the terraform file. 

```

Error: Missing required argument

  on cdk.tf.json line 37, in resource.aws_cloudwatch_event_rule.acmeexample_testeventbridgerule_9743ED72:
  37:       }

"event_pattern": one of `event_pattern,schedule_expression` must be specified
```